interface/patient_file/pos_checkout.php

```diff
@@ -720,122 +710,141 @@ function markTaxes($taxrates)
     // and OK and Cancel buttons.
     ?>
 <!DOCTYPE html>
+    <html>
     <head>
-            <?php Header::setupHeader(['datetime-picker']);?>
```

Comment:
I fix the missing <html> tag in here, 
but I'm not sure maybe need the @bradymiller to confirm.

Also, can you guide me to go to the Receipt for Payment page?
seems like this file contains two different pages.

Reply:
hi @stu01509 ,
Just click Save on this form, then you will be directed to the Receipt for Payment page:

![image](https://user-images.githubusercontent.com/278968/91952039-a9c95080-ecb5-11ea-9e51-9a856392770a.png)